### PR TITLE
#835 - added debian-adapter dependency and test

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #835 
Added debian adapter and first IT case.

### CODE REVIEW

File: src/main/java/com/artipie/SliceFromConfig.java [hunk 1]
```diff
@@ -299,6 +301,12 @@ static Slice build(
             case "docker-proxy":
                 slice = new DockerProxy(SliceFromConfig.HTTP, standalone, cfg, permissions, auth);
                 break;
+            case "deb":
+                slice = trimIfNotStandalone(
+                    settings, standalone,
+                    new LoggingSlice(new DebianSlice(cfg.storage(), permissions, auth, cfg.name()))
```
Comment (olegmoz) [inline src/main/java/com/artipie/SliceFromConfig.java:307]: @olenagerasimova I don't think we should wrap `DebianSlice` with `LoggingSlice` here, logging is enabled & configured on other level

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, for noticing, fixed

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#836` is not assigned, can't get performer

**Pull request merged**